frontend/lib/ui/Badge/Badge.stories.tsx

```diff
@@ -1,13 +1,26 @@
 import type { Story } from "@ladle/react";
 
+import type { PollingStationStatus } from "@kiesraad/api";
+
 import { Badge, BadgeProps } from "./Badge";
 
+const badgeTypes: PollingStationStatus[] = [
+  "not_started",
+  "first_entry_in_progress",
+  "first_entry_unfinished",
+  "second_entry",
+  "second_entry_in_progress",
+  "second_entry_unfinished",
+  "first_second_entry_different",
+  "definitive",
+];
+
 export const AllBadges: Story = () => {
   return (
     <>
-      <Badge type="definitive" />
-      <Badge type="not_started" />
-      <Badge type="first_entry_in_progress" showIcon />
+      {badgeTypes.map((type) => (
+        <Badge key={type} type={type} showIcon />
```

Comment:
If we wrap this in a `<div id={type}>`, then we can get rid of the `.nth()`s in the e2e test. And we can expand the e2e test to check for `.toHaveAttribute('data-icon', 'IconPencil');` where relevant.